sites/all/modules/hid/hid_profiles/hid_profiles.module

```diff
@@ -729,6 +729,14 @@ function hid_profiles_block_view($delta = '') {
 </div>  </div>';
       break;
     case 'hid_profiles_filters':
+      $node = node_load(arg(1));
+      $operation = $node;
```

Comment:
This is more for clarity reasons: I don't want to use the $operation variable only, because the node loaded may actually be a cluster. Would prefer to keep that part the way it is, even though it seems redundant at first...